src/Controller/Admin/ModuleRecommendedController.php

```diff
@@ -57,7 +57,7 @@ public function indexAction(): JsonResponse
             $tabClassName = $this->requestStack->getCurrentRequest()->get('tabClassName');
             $tab = $tabCollection->getTab($tabClassName);
             $context = $this->get('mbo.cdc.context_builder')->getRecommendedModulesContext($tab);
-            $context['recommendation_format'] = $tab->shouldDisplayAfterContent() ? 'card' : 'modal';
+            $context['recommendation_format'] = $this->requestStack->getCurrentRequest()->get('recommendation_format');
```

Comment:
Do the request send this param?

Reply:
I just have my answer : https://github.com/PrestaShopCorp/ps_mbo/pull/379/files#diff-93a8a6fd7a02a64cbdb1412e0298f3b57f7da7ec475cebf35e9ed819d01f0c6dR198